.circleci/config.yml

```diff
@@ -76,15 +76,14 @@ jobs:
 
   release:
     docker:
-      - image: circleci/golang:1.10
+      - image: circleci/golang:1.10.1
     working_directory: /go/src/github.com/roboll/helmfile
     steps:
     - checkout
     - setup_remote_docker
     - run:
         command: |
           docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io
-          make tools
```

Comment:
`make tools` seems to be still required for `make release`. It depends on `ghr` for managing gh releases.

Reply:
Good point: https://github.com/roboll/helmfile/pull/129